docs/packages/pkg/Boost.rst

```diff
@@ -89,6 +90,44 @@ For example:
 -  `boost.iostreams
    options <http://www.boost.org/doc/libs/1_57_0/libs/iostreams/doc/index.html?path=7>`__
 
+Boost::Math
```

Comment:
This is a little bit confusing. There is no imported library `Boost::Math`, C++ namespace is `boost::`. Let it be just `Math` then.